src/com/oltpbenchmark/benchmarks/seats/SEATSLoader.java

```diff
@@ -561,44 +615,37 @@ public void loadTable(Connection conn, Table catalog_tbl, Iterable<Object[]> ite
                             insert_stmt.setNull(i + 1, sqlTypes[i]);
                         }
                     } catch (SQLDataException ex) {
-                        LOG.error("INVALID " + catalog_tbl.getName() + " TUPLE: " + Arrays.toString(tuple));
-                        throw new RuntimeException("Failed to set value for " + catalog_tbl.getColumn(i).fullName(), ex);
+                        LOG.error("INVALID {} TUPLE: {}", catalog_tbl.getName(), Arrays.toString(tuple));
+                        throw new RuntimeException("Failed to set value for " + catalog_tbl.getColumn(i).getName(), ex);
                     }
-                } // FOR
+                }
                 insert_stmt.addBatch();
                 row_idx++;
 
                 if (++row_batch >= batch_size) {
                     LOG.debug(String.format("Loading %s batch [total=%d]", catalog_tbl.getName(), row_idx));
                     insert_stmt.executeBatch();
-                    conn.commit();
                     insert_stmt.clearBatch();
                     row_batch = 0;
                 }
 
-            } // FOR
+            }
 
             if (row_batch > 0) {
                 insert_stmt.executeBatch();
-                conn.commit();
             }
-            insert_stmt.close();
+
         } catch (Exception ex) {
             throw new RuntimeException("Failed to load table " + catalog_tbl.getName(), ex);
         }
 
-        if (is_airport) {
-            assert (this.profile.getAirportCount() == row_idx) : String.format("%d != %d", this.profile.getAirportCount(), row_idx);
-        }
-
         // Record the number of tuples that we loaded for this table in the
         // profile
-        if (catalog_tbl.getName().equals(SEATSConstants.TABLENAME_RESERVATION)) {
+        if (catalog_tbl.getName().equalsIgnoreCase(SEATSConstants.TABLENAME_RESERVATION)) {
```

Comment:
oh, interesting, wonder if this could be cause for issues elsewhere sometime

Reply:
In general case sensitivity is something that this PR seems to address in its own way